### PR TITLE
Adding expand parameter bundle testing to report runner

### PR DESCRIPTION
**Pull request by Chen2x**
Created a new test case for the drs object info test. This test adds an expand parameter to the request for the DRS object with the expectation that a bundle is returned if it is expandable. Added new file schema/v1.2.0/bundle_object.json which is used as the json schema to match the expected returned bundle. It is modelled after the drs object schema with a required 'contents' field.

### CODE REVIEW

**Comment by yash-puligundla** [comment]
@Chen2x could you please add a title and high level summary of what is included in this PR?

**Comment by yash-puligundla** [comment]
@Chen2x can you please describe what schema/v1.2.0/bundle_object.json is? 
Also, you must have missed deleting the file schema/v1.2.0/drs_bundle_backup.json

**Comment by Chen2x** [comment]
I have removed the backup file and added a description for bundle_object.json in the description.

File: compliance_suite/report_runner.py [hunk 1]
```diff
@@ -195,13 +204,52 @@ def test_drs_object_info(
...
+    # Skip test if contents field is not available
+    if is_bundle:
+        ### CASE: response expand bundle
+        add_test_case_common(
+            test_object = drs_object_test,
+            case_type = "expand_bundle",
+            case_name = "DRS Access expand bundle validation",
+            case_description = f"Validate DRS bundle when expand = True",
+            response = response,
+            schema_name = os.path.join(schema_dir, 'drs_bundle.json'))
```
Comment (yash-puligundla) [inline compliance_suite/report_runner.py:234]: Add a constant `DRS_BUNDLE_SCHEMA = "drs_bundle.json"` to compliance_suite/constants.py and use the constant here

**Review by yash-puligundla — APPROVED** [review]
Looks great!!

**Pull request merged**